bot.py

```diff
@@ -0,0 +1,19 @@
+import os
+from e_edu_bot import EEduBot
+import threading
+
+token = os.environ.get("DISCORD_TOKEN")
+
+if token is None or len(token) < 5:
```

Comment:
Check if file has been execute or imported:
```python
if __name__ == "__main__":
    pass
    # replace pass with your code
```

Reply:
? the file bot.py should not be imported. it only starts the bot